views/partials/header.hbs

```diff
@@ -1,13 +1,11 @@
 <div class="header">
-  <a href="/"><h1 class="text-center naz-events">Nazareth Events</h1></a>
-  <a href="/add-event" class="glyphicon glyphicon-plus"></a>
+  <a href="/add-event"><i class="fa fa-plus	plus" alt="add event"></i>
+  <a href="/"><span class="naz-events">Nazareth Events</span></a>
```

Comment:
Same here, no reason to have a `span` inside the `a` tag, just put the text in the `a` tag